test/index.js

```diff
@@ -16,15 +16,15 @@ describe("Check Plugin's exported object", () => {
 		SoliumSecurityPlugin.should.have.ownProperty("meta");
 		SoliumSecurityPlugin.should.have.ownProperty("rules");
 
-		const {meta, rules} = SoliumSecurityPlugin;
+		const { meta, rules } = SoliumSecurityPlugin;
 
 		meta.should.type("object");
 		meta.should.have.size(1);
 		meta.should.have.ownProperty("description");
 		meta.description.should.be.type("string");
 
 		rules.should.be.type("object");
-		rules.should.have.size(8);
+		rules.should.have.size(9);
```

Comment:
Need to be careful here; if another rule is merged before this one, we need to increment this.

Reply:
Yes I'm aware of this. I'll take care of resolving the conflicts!